### ISSUE TITLE
Version number of PEGTL should be 3.0.0, not 2020-12-07

### ISSUE THREAD

**Issue by d-frey** [microsoft/vcpkg#15052]
https://github.com/microsoft/vcpkg/blob/c30903782c88967652178486a5f3e490fb2e8143/ports/pegtl/CONTROL#L2

In the above, the 2020-12-07 should be replaced with 3.0.0

**Comment by JonLiu1993** [comment]
@d-frey ,Thanks for posting this issue,Version:3.0.0，Version 3.0 is the latest stable version of pegtl, and 2020-12-07 is the latest modified version submitted by pegtl on the same day. This practice is in compliance with vcpkg regulations in vcpkg, you can refer to here:[manifest-files.md#version-fields](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/manifest-files.md#version-fields)
````
For rolling-release ports, we use the date that the commit was accessed by you, formatted as YYYY-MM-DD. Stated another way: if someone had a time machine and went to that date, they would see this commit as the latest master.

For example, given:

The latest commit was made on 2019-04-19
The current version string is 2019-02-14
Today's date is 2019-06-01.
Then if you update the source version today, you should give it version 2019-06-01.
````

**Comment by d-frey** [comment]
I see, although it is slightly confusing as there was a "3.0.0-pre" previously and the commit hash of the "2020-12-07" is from 2020-12-06. Anyways, I'm fine with it and I understand that @wravery probably wanted the fallback for `<filesystem>` in, hence not using a tagged release. We will soon release a new version 3.1.0 with all those and more improvements, so that Vcpkg can than have an "official" version :) Thanks!

**Comment by wravery** [comment]
Yes, that was a maintainer edit that went in just before the PR was merged. I originally put 3.0.0 on the version number and used that release tag, but then it encountered the `<filesystem>` change which blocked some of the vcpkg CI jobs, and I had to pull in additional commits after that to get matching/working builds of PEGTL and cppgraphqlgen that passed CI for vcpkg.

It is a little confusing to have the date there instead of a version number, or even a version number with a suffix. But I noticed a lot of vcpkg ports seem to be using a date now instead of version numbers or commit hashes. It looks like maybe that guidance has been updated since I last looked at that doc, or maybe I missed that distinction between versioned releases and rolling releases. Once 3.1.0 is available I think we can snap back to a release version number instead of the rolling release date. 👍🏼
